eureka-client/src/test/java/com/netflix/eureka2/client/registration/RegistrationHandlerImplTest.java

```diff
@@ -64,17 +63,10 @@ public void testRegistersClientsOverDifferentChannel() throws Exception {
     @Test
     public void testConvertsRegistrationToUpdateIfAlreadyRegistered() throws Exception {
```

Comment:
Better test name? We do not convert to updates, but test multiple registrations for the same service.